src/exchange/exchange.provider.ts

```diff
@@ -82,6 +81,10 @@ export class ExchangeProvider {
     return this.exchanges.get(exchangeName).priceToPrecision(symbol, price)
   }
 
+  public limits(exchangeName: string, symbol: string) {
```

Comment:
Nitpicking here, but I'd prefer the name `getLimits`. Having a member named just `limits` would be okay if it was a property accessor.

Reply:
agreed. I have renamed it.